src/l2/input_parser.star

```diff
@@ -150,6 +153,30 @@ def _parse_instance(l2_args, l2_name, l2_id_generator, registry):
         registry=registry,
     )
 
+    # We add the flashblocks params
+    l2_params["flashblocks_websocket_proxy_params"] = _flashblocks_input_parser.parse(
+        websocket_proxy_args=l2_params["flashblocks_websocket_proxy_params"],
+        network_params=l2_params["network_params"],
+        registry=registry,
+    )
+
+    # WARNING: We can't have another participant with the same name
+    # TODO: We should check whether any participant has the same name
```

Comment:
I'd make a ticket or a PR if you're adding a TODO, otherwise they tend to never get untodoed